tests/models/test_dagrun.py

```diff
@@ -874,3 +875,20 @@ def test_verify_integrity_task_start_date(Stats_incr, session, run_type, expecte
     assert len(tis) == expected_tis
 
     Stats_incr.assert_called_with('task_instance_created-DummyOperator', expected_tis)
+
+
+@pytest.mark.xfail(reason="TODO: Expand mapped literals at verify_integrity time!")
```

Comment:
I'm not sure this is actually a good idea -- although we _could_ put it here, that puts more work in the core scheduler loop so I think we could reasonably delay this to the mini scheduler in upstream task.

Thoughts?

Reply:
It won't _only_ be in the mini scheduler run, there will still be a "expansion of last resort" in the scheduler. I guess the difference is do we want to do the expansion eagerly at DagRun creation time, when it could possibly be done in another process (the LocalTaskJob).

It's probably going to be quite rare in practice that maps will be literals, so I think it's not even the cost to check this here, given that it's so unlikely it will do anything useful.